crud/common/utils.lua

```diff
@@ -181,6 +181,13 @@ local function determine_enabled_features()
 
     -- since Tarantool 2.4
     enabled_tarantool_features.uuids = major >= 2 and (minor > 4 or minor == 4 and patch >= 1)
+
+    -- since Tarantool 1.10
+    enabled_tarantool_features.jsonpath_filters = major >= 2 or (major >= 1 and minor >= 10)
```

Comment:
It doesn't make sense since crud supports only 1.10+.

You couldn't run crud e.g. on Tarantool 1.9 even before this patch.